src/themes/default/stylesheets/main.scss

```diff
@@ -426,6 +426,7 @@ $background-arguments: #fafbfc;
   }
 
   .deprecation-reason {
+    word-break: break-word;
```

Comment:
The dep reasons were also causing issues in mobile, now they will break if things are too small. Seems like the browser doesnt like to break on the `::before` elements, so this is the deal:

<img width="318" alt="Screen Shot 2022-04-05 at 4 48 00 PM" src="https://user-images.githubusercontent.com/69169/161869082-dadd30f1-55bf-4d12-96bf-0371426ebcc5.png">